library/entity/entity_extensions/src/testmod/resources/quilt.mod.json

```
@@ -0,0 +1,27 @@
+{
+  "schema_version": 1,
+  "quilt_loader": {
+    "group": "org.quiltmc.qsl.entity",
+    "id": "quilt_entity_testmod",
+    "version": "1.0.0",
+    "metadata": {
+      "name": "Quilt Entity API Test Mod",
+      "license": "Apache-2.0"
+    },
+    "intermediate_mappings": "net.fabricmc:intermediary",
+    "load_type": "always",
+    "entrypoints": {
+      "init": [
+        "org.quiltmc.qsl.entity.test.villager.VillagerTypeTest1",
+        "org.quiltmc.qsl.entity.test.villagerVillagerTypeTest2",
```

uh, have you tested this PR?
```suggestion
        "org.quiltmc.qsl.entity.test.villager.VillagerTypeTest2",
```